packages/neptune-extension/src/common/components/input/Input.tsx

```diff
@@ -0,0 +1,64 @@
+// Lib
+import React from 'react';
+import { bemBlock } from 'common/utils/bem';
+
+// Module
+import './Input.less';
+
+interface InputProps {
+  className?: string
+  disabled?: boolean
+  error?: boolean
+  selectOnFocus?: boolean
+  variant?: 'small' | 'big'
```

Comment:
This is not implemented in styles and there is no case for such distincion.
However there should be a support for textarea.

Reply:
Textarea is different component.
We don't need textarea in extension as far as I know.
ENVs code can be displayed in custom component that is not editable.

Reply:
I'll remove variant